source/includes/project.java

```diff
@@ -0,0 +1,35 @@
+# start-project-include
```

Comment:
S: make this file into a complete java file. Also, I could be wrong but I am pretty sure java comments are written with `//` characters

Reply:
Hey, what do you mean by a complete java file? This is following the lead of other standardized pages, where we give non-runnable code snippets.

Reply:
I see. It's not my preference as it makes it more difficult to test examples, but I'll leave it up to you to decide what the standard should be.